go/libkb/saltpack.go

```diff
@@ -62,12 +62,11 @@ type naclBoxSecretKey NaclDHKeyPair
 var _ saltpack.BoxSecretKey = naclBoxSecretKey{}
 
 func (n naclBoxSecretKey) Box(
-	receiver saltpack.BoxPublicKey, nonce *saltpack.Nonce, msg []byte) (
-	[]byte, error) {
+	receiver saltpack.BoxPublicKey, nonce *saltpack.Nonce, msg []byte) []byte {
 	ret := box.Seal([]byte{}, msg, (*[24]byte)(nonce),
 		(*[32]byte)(receiver.ToRawBoxKeyPointer()),
 		(*[32]byte)(n.Private))
-	return ret, nil
+	return ret
```

Comment:
This never returns an error, so I've removed the err value from the interface. We did similar cleanup on the SecretBox side of things earlier.